pkgs/tools/archivers/gnutar/default.nix

```diff
@@ -1,4 +1,4 @@
-{ lib, stdenv, fetchurl, autoreconfHook, acl, libintl }:
+{ lib, stdenv, fetchurl, makeWrapper, autoreconfHook, acl, libintl }:
```

Comment:
Shouldn't we use `makeBinaryWrapper` instead for better compatibility with Darwin archs?